### PR TITLE
Modify And Delete Org

### PR DESCRIPTION
**Pull request by Colin-Lane**
@leogrim @ryanpbrewster

### CODE REVIEW

File: kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationCommander.scala [hunk 1]
```diff
@@ -85,7 +85,7 @@ class OrganizationCommanderImpl @Inject() (
     val badName = modifications.name.exists(_.isEmpty)
     val badBasePermissions = modifications.basePermissions.exists { bps =>
       // Are there any members that can't even see the organization?
-      OrganizationRole.all exists { role => !bps.forRole(role).contains(VIEW_ORGANIZATION) }
+      OrganizationRole.all exists { role => !(bps.permissionsMap.contains(Some(role)) && bps.forRole(role).contains(VIEW_ORGANIZATION)) }
```
Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationCommander.scala:88]: No longer necessary. We impose this condition by construction.

Comment (Colin-Lane) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationCommander.scala:88]: Not true, these permissions come from the modification request. The request is not validated until this step.

Comment (ryanpbrewster) [inline kifi-backend/modules/shoebox/app/com/keepit/commanders/OrganizationCommander.scala:88]: Nevermind, this is probably fine.

**Comment by ryanpbrewster** [comment]
:+1:

**Pull request merged**